.ci/openshift-ci/buildall_install.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Build kata-containers and install into a given destination directory
+#
+
+set -e
+
+if [ -z "$1" ];then
+	echo "Usage: $0 path/to/install/dir"
+	exit 1
+fi
+
+export DESTDIR=$1
+echo "Build and install kata-containers at: ${DESTDIR}"
+
+export PATH=${GOPATH}/bin:$PATH
+
+cidir=$(dirname "$0")/..
+source /etc/os-release || source /usr/lib/os-release
+source "${cidir}/lib.sh"
+
+# The scripts rely on sudo which is not installed in the build environment.
+yum install -y sudo
+"${cidir}/setup_env_centos.sh" default
+
+# The build root container has already golang installed, let's remove it
+# so that it will use the version required by kata.
+yum remove -y golang
+"${cidir}/install_go.sh" -p -f
+
+# Let's ensure scripts don't try things with Podman.
+export TEST_CGROUPSV2=false
+
+# Use the QEMU experimental (QEMU with virtiofs).
+export experimental_qemu=true
+export experimental_kernel=true
+
+# Install rootfs image which is supported with QEMU experimental.
+export TEST_INITRD=no
+
+# Configure to use vsock.
+# TODO: install_runtime.sh will try to load the vsock module and the script
```

Comment:
Please can you add a link to a GH issue here:

```sh
# TODO: Need to do blah. See: https://github.com/kata-containers/tests/issues/XXX
```

Reply:
done